Tests/UnitTests/Purchasing/ErrorCodeTests.swift

```diff
@@ -181,3 +181,100 @@ class ErrorCodeTests: TestCase {
     }
 
 }
+
+import StoreKit
+
+class ErrorUtilsTests: TestCase {
```

Comment:
Hmm would it be better moving these to their own file?

Reply:
Oh yeah let me do that.

Reply:
Done